### PR TITLE
:zap: Try to improve performance by using channels

### PR DESCRIPTION
**Pull request by WtfJoke**
Locally the use of channels is for some reasons slower... I wasnt able to test it on the lambda because my internet is way too bad atm (I wasnt able to update the layer even after multiple retries).

Maybe its just the overhead of using a channel 🤷 

FYI: @RaphaelManke

### CODE REVIEW

File: extension/secrets.go [hunk 1]
```diff
@@ -25,7 +25,9 @@ func LoadSecrets() {
 	fmt.Println("Current timestamp:", time.Now().UnixMilli())
 
 	secretIds := readSecretIdsFromEnvironmentWhenStartsWithSecret()
-	secrets := getSecretValuesFromListOfSecretIds(secretIds)
+	secretsChannel := make(chan Secret, len(secretIds))
+	go getSecretValuesFromListOfSecretIds(secretIds, secretsChannel)
```
Comment (RaphaelManke) [inline extension/secrets.go]: I don't think the "go" command is correct here? As far as I understood it the go is the fan out. I think what happens here is that the get secretValues function is "async" but the secret values are fetched in sequence ?

File: extension/secrets.go [hunk 2]
```diff
@@ -43,17 +45,17 @@ func readSecretIdsFromEnvironmentWhenStartsWithSecret() []string {
...
-func getSecretValuesFromListOfSecretIds(secretIds []string) []Secret {
-	var secrets []Secret
+func getSecretValuesFromListOfSecretIds(secretIds []string, secretsChannel chan Secret) {
 	cfg, err := config.LoadDefaultConfig(context.TODO())
 	checkError(err)
 	client := secretsmanager.NewFromConfig(cfg)
 
 	for _, secretId := range secretIds {
+		log.Print("Fetch secret: "+secretId+" on ", time.Now().UnixMilli())
 		secret := getSecretValue(client, secretId)
```
Comment (RaphaelManke) [inline extension/secrets.go]: I think the go command should be here so that all the getSecret calls run in parralel.

**Comment by RaphaelManke** [comment]
Should we remove some of the log messages? Right now it is very verbose

File: extension/secrets.go (hunk 1 above)
Comment (WtfJoke) [reply extension/secrets.go]: Im not sure either... could be. At least it felt like it was still worked in sequence. Also im not sure if it now works allright 😅

**Comment by WtfJoke** [comment]
I've removed some of the log messages. Now it feels more normal. Feel free to merge if you think its allright

**Pull request merged**